packages/api/db/migration/20240207214919_create_animal_union_batch_internal_id_view_modify_animal_batch_tables.js

```diff
@@ -0,0 +1,83 @@
+/*
+ *  Copyright 2024 LiteFarm.org
+ *  This file is part of LiteFarm.
+ *
+ *  LiteFarm is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  LiteFarm is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the
+ *  GNU General Public License for more details, see <https://www.gnu.org/licenses/>.
+ */
+
+/**
+ * @param { import("knex").Knex } knex
+ * @returns { Promise<void> }
+ */
+export const up = async function (knex) {
+  try {
+    await knex.schema.alterTable('animal', (table) => {
+      table.string('photo_url');
+      table.dropChecks(['name_identifier_check']);
+    });
+
+    await knex.schema.alterTable('animal_batch', (table) => {
+      table.string('photo_url');
+      table.string('name').nullable().alter();
+    });
+
+    // Create function to assign internal_identifier to a record (NEW) to be inserted
+    await knex.raw(`
+      CREATE VIEW animal_union_batch_internal_identifier AS
```

Comment:
Copilot suggested (but it makes sense to me!) that the view might be named with `_view` to make it clearer in the future that this is a view and not a table; e.g. `animal_union_batch_id_view`.

I was thinking of that in particular looking at the model, which has no indication that this is not full table.